.github/workflows/scan-python.yaml

```diff
@@ -9,6 +9,16 @@ on:
         description: |
           Packages to install with apt when building the wheel or scanning with trivy
           This can be useful if creating a virtual environment has extra build-deps.
+      uv-export:
+        type: boolean
+        default: true
+        description: |
+          Whether to export requirements files from uv. Defaults to true.
+      uv-sync-extra-args:
+        type: string
+        default: "--no-dev"
```

Comment:
how come this is the default?

Reply:
Because we probably don't want to do a security scan on development dependencies, but I don't want to enforce it because we may end up with a project that does want that.

Reply:
shouldn't we do security scans on dev dependencies though? Some vulnerabilities might exfiltrate secrets from CI, etc

Reply:
The initial thought was that this is more a renovate/dependabot kind of job, but you've convinced me on the practicality vs. purity front.